presto-accumulo/src/main/java/com/facebook/presto/accumulo/model/RowSchema.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo.model;
+
+import com.facebook.presto.spi.PrestoException;
+import com.facebook.presto.spi.type.Type;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static com.facebook.presto.accumulo.AccumuloErrorCode.COLUMN_NOT_FOUND;
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+
+/**
+ * Class to define the schema of a Row, stored as a list of {@link AccumuloColumnHandle}.
+ */
+public class RowSchema
+{
+    private List<AccumuloColumnHandle> columns = new ArrayList<>();
+
+    /**
+     * Gets a new instance of RowSchema.
+     *
+     * @return A new RowSchema
+     */
+    public static RowSchema newRowSchema()
+    {
+        return new RowSchema();
+    }
+
+    /**
+     * Adds the Row ID column to the schema
+     *
+     * @param name Name of the row ID column
+     * @param type Presto type of the column
+     * @return this, for chaining
+     */
+    public RowSchema addRowId(String name, Type type)
+    {
+        columns.add(new AccumuloColumnHandle(name, null, null, type, columns.size(),
+                "Accumulo row ID", false));
```

Comment:
Just for the users when the describe the table to see how the Presto columns map to the Accumulo columns.  Otherwise they would have to get the JSON metadata from ZooKeeper for a refresher.